lib/template_handlers/erb.rb

```diff
@@ -0,0 +1,46 @@
+module TemplateHandlers
+  class ERB < ActionView::Template::Handlers::ERB
+    CONTENT_DIR = Rails.root.join("app/views/content").freeze
+
+    def call(template, source = nil)
+      # inspect objects to Ruby strings which can be eval'd
+      %(@sitemap = #{self.class.sitemap.inspect}; #{super}.html_safe)
+    end
+
+    class << self
+      attr_writer :sitemap
+
+      def sitemap
+        @sitemap ||= build_sitemap
+      end
+
+      def build_sitemap
+        resources = Dir.glob(content_pattern).map do |file|
+          {
+            path: path(file),
+            front_matter: front_matter(file),
+          }
+        end
+
+        { resources: resources }
```

Comment:
Resources feels like quite an ambiguous name? Unless its part of a bigger picture this may make more sense as something like specific like `markdown_pages`, or `markdown_content`.

Reply:
Agreed, I took the naming from the Middleman sitemap but it's not a good fit or representation of what this is for us. I'll merge this down then follow it up with another PR to rename it as I want to get the changes out early doors to avoid any disruption.